examples/with-iron-session-typescript/.env.development

```diff
@@ -0,0 +1,6 @@
+# ⚠️ The SECRET_COOKIE_PASSWORD should never be inside your repository directly, it's here only to ease
+# the example deployment
+# For local development, you should store it inside a `.env.local` gitignored file
+# See https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables
+
+SECRET_COOKIE_PASSWORD=2gyZ3GDw3LHZQKDhPmPDL3sjREVRXPr8
```

Comment:
Let's just update the main example to TS instead of having multiple - I think that's the direction we'll be headed in the future, teaching TS first.

Reply:
Thanks, I just replaced the with-iron-session example with the TypeScript one.